packages/wdio-sync/src/wrapCommand.js

```diff
@@ -12,13 +12,19 @@ import { sanitizeErrorMessage } from './utils'
  * @return {Function}   actual wrapped function
  */
 export default function wrapCommand (commandName, fn) {
+    let stackError
     return function wrapCommandFn (...args) {
+        // save error for getting full stack in case of failure
+        // should be before any async calls
+        if (!stackError) {
+            stackError = new Error()
```

Comment:
wouldn’t this mean that the error would be always the same?

Reply:
the error - no, but the stack trace yes in some cases, need to fix it